knowledge_repo/config_defaults.py

```diff
@@ -53,6 +53,10 @@ def web_uri(path=None):
     return None
 
 
+# Prefixes of repositories that can be edited via the web editor UI
+# Defaults to no prefixes allowed. If None, all prefixes editable via the UI.
+web_editor_prefixes = []
```

Comment:
Shouldn't this key be specified in the server configuration, not the repository configuration? If not, then we need to propagate this configuration into methods on the `KnowledgeRepository` class and delegate in the `MetaKnowledgeRepository` class.